src/util.jl

```diff
@@ -206,3 +206,21 @@ end
 function type2string(type)
     return string(nameof(typeof(type)))
 end
+
+function compute_git_hash()
```

Comment:
Since this is no numerical stuff and just util stuff, I moved it to `util.jl` and removed the almost empty `file_system.jl`.